<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseMainPage</class>
 <widget class="QMainWindow" name="BaseMainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,8,1">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="BottomWidget" name="bottomWidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CBasePage" name="midWidget" native="true">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="TopWidget" name="topWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </item>
   </layout>
   <zorder>topWidget</zorder>
   <zorder>bottomWidget</zorder>
   <zorder>midWidget</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CBasePage</class>
   <extends>QWidget</extends>
   <header>BasePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BottomWidget</class>
   <extends>QWidget</extends>
   <header>bottomwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TopWidget</class>
   <extends>QWidget</extends>
   <header>topwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
